packages/electron-redux/src/middleware/__tests__/forwardToMain.js

```diff
@@ -9,7 +9,7 @@ describe('forwardToMain', () => {
     validateAction.mockReturnValue(true);
   });
 
-  it('should pass an action through if it doesn\'t pass validation (FSA)', () => {
+  it("should pass an action through if it doesn't pass validation (FSA)", () => {
```

Comment:
Did you skip the pre-commit hook? `prettier-eslint` shouldn't allow this

Reply:
It's the way prettier formats it. 

```
❯ npx prettier --single-quote packages/electron-redux/src/middleware/__tests__/forwardToMain.js | grep it
import forwardToMain, { forwardToMainWithParams } from '../forwardToMain';
  it("should pass an action through if it doesn't pass validation (FSA)", () => {
    expect(next).toHaveBeenCalledWith(action);
  it('should pass an action through if it starts with @@', () => {
    expect(next).toHaveBeenCalledWith(action);
```

I believe you've not updated these tests since you set up lint-staged.